packages/ingest-mongodb-public/src/sources/mongodb-university/makeUniversityPages.test.ts

```diff
@@ -89,3 +90,53 @@ describe("convertVideoTranscriptFromSrtToTxt()", () => {
     expect(convertedTranscript).toEqual(txtTranscript);
   });
 });
+
+describe("generateContentDescriptionMarkdown()", () => {
+  const expected_markdown = `# Atlas Search
+This course is designed to help you use Atlas Search in your applications for a seamless full-text search experience. You’ll learn how to write Atlas Search queries and manage search indexes by using the Atlas UI, CLI, and MongoDB Shell. Then you’ll build on this foundation by writing advanced search queries that use the compound operator, nesting, and features such as fuzzy search and autocomplete. 
+
+
+## Managing Atlas Search Indexes
+45 Minutes
+Learn how to manage Atlas Search indexes with the Atlas UI, Atlas CLI, and MongoDB Shell.
+[View Details](https://learn.mongodb.com/courses/managing-atlas-search-indexes)
+
+
+## Autocomplete with Atlas Search
+1.5 Hours
+Learn how to enhance user experience by adding the autocomplete functionality to Atlas Search indexes. As one of the most common features of a search platform, autocomplete is easy to include in any application using Atlas Search.  
+[View Details](https://learn.mongodb.com/courses/autocomplete-with-atlas-search)
+
+
+## Introduction to Atlas Search
+2.25 Hours
+Learn how to use Atlas Search to build search functionality for your application. You'll learn how to define a search index, create a search query, and view search metadata. 
+[View Details](https://learn.mongodb.com/courses/introduction-to-atlas-search)
+
+
+## Analyzers in Atlas Search
+1.75 Hours
+Learn all about analyzers in Atlas Search and the crucial role they play in making search results relevant and accurate.  
+[View Details](https://learn.mongodb.com/courses/analyzers-in-atlas-search)
+
+
+## Advanced Queries with Atlas Search
+3 Hours
+Learn how to write Atlas Search queries using features such as the compound operator, sorting customization, and fuzzy search. 
+[View Details](https://learn.mongodb.com/courses/advanced-queries-with-atlas-search)
+
+`;
```

Comment:
```suggestion
[View Details](https://learn.mongodb.com/courses/advanced-queries-with-atlas-search)`;
```

update code to remove trailing white space